pkg/controller/smart-clone-controller.go

```diff
@@ -356,6 +356,7 @@ func newPvcFromSnapshot(snapshot *snapshotv1.VolumeSnapshot, targetPvcSpec *core
 				AnnRunningConditionMessage: cloneComplete,
 				AnnRunningConditionReason:  "Completed",
 				annSmartCloneSnapshot:      key,
+				AnnPodPhase:                string(corev1.PodSucceeded),
```

Comment:
Maybe add a comment here that we are artifically adding this pod phase on a smart clone where no pods are involved. My first reaction when seeing this was why are pods involved in the smart clone controller.

After think I understand that you need this to determine if you can GC the DV, seems odd though.

Reply:
Adding comment. The alternative is an additional PVC annotation for all DV flows, which is even more odd...